Source/Devices/HarpSyncInput.cpp

```diff
@@ -29,6 +29,8 @@ HarpSyncInput::HarpSyncInput (std::string name, std::string hubName, const oni_d
 {
     setEnabled (false);
 
+    const ContinuousChannel::InputRange harpTimeRange { 0.0f, 1000.0f };
```

Comment:
What's the rationale for making this only go up to 1000? Won't it max out after ~16 minutes?